src/cmdline/mod.rs

```diff
@@ -248,18 +252,24 @@ impl Cmdline {
         Ok(())
     }
 
-    /// Returns the string representation of the command line without the nul terminator.
+    /// Returns a C compatible representation of the command line (null terminated)
```

Comment:
I don't think that we should write here null terminated because it's just an abstraction. To get the null terminated bytes you need to call a specific function.